### PR TITLE
enable users to refresh options of combobox (from Azure) in case of data inconsistency.

### PR DESCRIPTION
**Pull request by wangmingliang-ms**
What does this implement/fix? Explain your changes.
---------------------------------------------------
* add default `refresh` extension on all comboboxes.
* clear cache(if have) first when refreshing.

Does this close any currently open issues?
------------------------------------------
[AB#1974371](https://dev.azure.com/mseng/a4d27ce2-a42d-4b71-8eef-78cee9a9728e/_workitems/edit/1974371): add refresh action on combobox component, so that users can always refresh options from remote server in case of inconsistence.

Any relevant logs, screenshots, error output, etc.?
-------------------------------------

Any other comments?
-------------------
…

Has this been tested?
---------------------------
- [ ] Tested

### CODE REVIEW

File: PluginsAndFeatures/azure-toolkit-for-intellij/azure-intellij-plugin-lib-java/src/main/java/com/microsoft/azure/toolkit/intellij/common/AzureArtifactComboBox.java [hunk 1]
```diff
@@ -67,21 +70,27 @@ public AzureArtifact getValue() {
...
     }
 
-    @Nullable
+    @Nonnull
     @Override
-    protected ExtendableTextComponent.Extension getExtension() {
-        return ExtendableTextComponent.Extension.create(AllIcons.General.OpenDisk, "Open file", this::onSelectFile);
+    protected List<Extension> getExtensions() {
+        final List<Extension> extensions = super.getExtensions();
+        final KeyStroke keyStroke = KeyStroke.getKeyStroke(KeyEvent.VK_INSERT, InputEvent.ALT_DOWN_MASK);
```
Comment (Flanker32) [inline PluginsAndFeatures/azure-toolkit-for-intellij/azure-intellij-plugin-lib-java/src/main/java/com/microsoft/azure/toolkit/intellij/common/AzureArtifactComboBox.java]: use `KeyStroke.getKeyStroke(KeyEvent.VK_ENTER, InputEvent.SHIFT_DOWN_MASK)` same as function core tools combo box?

File: PluginsAndFeatures/azure-toolkit-for-intellij/azure-intellij-plugin-database/src/main/java/com/microsoft/azure/toolkit/intellij/database/component/UsernameComboBox.java [hunk 2]
```diff
@@ -32,4 +36,10 @@ public void setServer(IDatabaseServer<?> server) {
     public boolean isRequired() {
         return true;
     }
+
+    @Nonnull
+    @Override
+    protected List<ExtendableTextComponent.Extension> getExtensions() {
```
Comment (Flanker32) [inline PluginsAndFeatures/azure-toolkit-for-intellij/azure-intellij-plugin-database/src/main/java/com/microsoft/azure/toolkit/intellij/database/component/UsernameComboBox.java]: shall we keep refresh for username combobox? Which may be affected by users' operation outside toolkit

Comment (wangmingliang-ms) [reply PluginsAndFeatures/azure-toolkit-for-intellij/azure-intellij-plugin-database/src/main/java/com/microsoft/azure/toolkit/intellij/database/component/UsernameComboBox.java]: fixed.

File: PluginsAndFeatures/azure-toolkit-for-intellij/azure-intellij-plugin-lib-java/src/main/java/com/microsoft/azure/toolkit/intellij/common/AzureArtifactComboBox.java (hunk 1 above)
Comment (wangmingliang-ms) [reply PluginsAndFeatures/azure-toolkit-for-intellij/azure-intellij-plugin-lib-java/src/main/java/com/microsoft/azure/toolkit/intellij/common/AzureArtifactComboBox.java]: fixed.

**Review by Flanker32 — APPROVED** [review]
LGTM

**Pull request merged**